docs/finality-provider-operation.md

```diff
@@ -810,48 +786,38 @@ This command should ask to
 `confirm transaction before signing and broadcasting [y/N]:` and output the
 transaction hash.
 
-### 6.2.4. Refund Finality Provider
-
-To support the gas costs associated with committing randomness, which are not
-refunded by the protocol, we recommend setting up a refunding flow.
-This involves periodically transferring rewards from a beneficiary key to the
-operational key used by the finality provider.
+### 6.3. Set Up Operation Key
 
-#### How to set up the refunding process
+Finality providers consume gas for operations with Babylon Genesis.
+Given that the cost of sending finality signatures is refunded automatically
+after success, the only cost of operating a finality provider is randomness
+commit, which is made periodically with very low gas cost.
 
-1. Ensure you have two keys:
-   * Beneficiary Key: Receives finality provider rewards.
-   * Operational Key: Used by the finality provider daemon to submit transactions.
+Therefore, it is recommended to use a separate key specifically for operations.
+This leaves key used for registration key isolated and only needed for claiming
```

Comment:
```suggestion
This leaves the key used for registration key isolated and only needed for claiming
```

Reply:
Is this true? I was under the impression that the following holds:

* Key used during registration **must** be the same as the one used for operations.
* However, the registration key can be used to set a separate withdraw address to a more secure keyring than the one used for ops.

From the text it looks like the register key has to be the secure one, while there's a gas-payment key. How are the two keys associated, with the Babylon node knowing that operations (e.g. randomness commits) are coming from the owner of the register key?

Reply:
Will reply offline.